tools/rebuildEM/main.cpp

```diff
@@ -76,6 +76,12 @@ int main(int argc, char** argv)
         }
     }
 
+    // MCOL-4685
+    std::cout << "The launch of mcsRebuildEM tool must be sanctioned by MariaDB support. "
+              << std::endl;
+    std::cout << "Requirement: all DBRoots must be on this node. " << std::endl;
+    std::cout << "Note: that the launch can break the cluster." << std::endl;
```

Comment:
@denis0x0D I might had missed the point but where does the tool asks for a confirmation before it begins?

Reply:
Oh, my fault, forgot to add the question about confirmation. Thanks.